src/base_object.h

```diff
@@ -36,7 +36,7 @@ class BaseObject {
  public:
   // Associates this object with `handle`. It uses the 0th internal field for
```

Comment:
Actually, the comment needs to be updated too :)

Reply:
Oh, I missed that. I'll update it. Thanks!